mitosheet/mitosheet/step_performers/graph_steps/graph.py

```diff
@@ -139,7 +148,7 @@ def execute(  # type: ignore
                     l=0,
                     r=0,
                     t=30,
-                    b=30,
+                    b=35, # This gives enough space so that the x axis label is not cutoff
```

Comment:
@aarondr77 we should test this stuff on windows before deploying! I'm fine to do this after we deploy on Test PyPi, so that we can avoid your environment setup issues.